release-notes/v0.9.0.md

```diff
@@ -0,0 +1,200 @@
+- [Syntax Mode](#syntax-mode)
+- [Java 14](#java-14-supported)
+- [Call Hierarchy](#call-hierarchy)
+- [Data Breakpoint](#data-breakpoint-supported)
+- [Code Actions](#more-code-actions)
+- [SonarLint](#sonarlint)
+- [Work with JAR Files Directly](#work-with-jar-files-directly)
+- [Create New Class from File Explorer](#create-new-class-from-file-explorer)
+- [MicroProfile](#microprofile)
+- [Others](#others)
+
+## Performance Improvements
+
+The Java Language Server (the language server) has improved its speed of ***startup***, and the speed to ***import*** Maven projects. The team has been committed, and here are what they delivered so far.
+
+### Code Navigation in 10 Seconds
+
+Tired of waiting on the language server to start? Now the wait is over. With the help of the new Syntax Server, you will be able to use a fair amount of features within 10 seconds:
+
+![Syntax Server Startup](v0.9.0/syntax-server-startup.gif)
+
+This is useful especially when you open a project for the first time. Instead of waiting for all the dependencies to be downloaded, you can use the features listed below:
+
+- Syntax Highlighting & Syntax Errors
+- Code Navigation (Go to Definition, etc.)
+- Documentation (Hover to See Javadoc)
+- Code Structures (Outline, Folding Range, etc.)
+
+If you are interested in the technical details behind this, check out the [Syntax Mode](#syntax-mode) section.
+
+### Download Maven Dependencies in Parallel
+
+If you are using Maven and your project depends on lots of artifacts from Maven Central, here's a piece of good news for you. The language server now downloads Maven dependencies in parallel, and it will save you lots of time. It happens automatically when you open a Maven project so you don't need to do anything. Enjoy!
+
+## Syntax Mode
+
+When you drag and drop some Java files in VS Code, you probably only want to view the code, but might not be interested in the type errors. The Syntax Mode came to rescue by hiding those type errors and only reporting syntax errors. Besides that, it also parses the code and provide structure info, so you can see outlines and fold/unfold code blocks. You can tell whether you are in syntax mode by checking out this message in the `PROBLEMS` view:
+
+![Syntax Mode Indicator](v0.9.0/syntax-mode-indicator.png)
+
+Syntax Mode is more than just syntax info. It allows you to browse the known types that are defined in your *local source*, or the *JDK*. You can navigate to the definition or the references to those types, and hover to view the documentation.
+
+However, if you do want to see all the type errors, you can switch between Syntax Mode and Full Mode by right-clicking on the warning message:
+
+![Syntax Mode Switch](v0.9.0/syntax-mode-switch.png)
+
+Syntax Mode is also the foundation of the Syntax Server. It is tolerant and is capable of working with missing types. So in Syntax Mode, you don't need to wait for the dependencies to be resolved.
```

Comment:
Is syntax mode **based on** syntax server? For regular users, I feel it would be nice to just introduce one concept to avoid confusion.

Reply:
It's the opposite, the Syntax Server is based on the Syntax Mode.

Reply:
good point. will do.